assets/src/edit-story/app/uploader/useUploader.js

```diff
@@ -0,0 +1,47 @@
+
+/**
+ * Internal dependencies
+ */
+/**
+ * WordPress dependencies
+ */
+import { useCallback } from '@wordpress/element';
+import { useAPI } from '../../app/api';
+import { useConfig } from '../config';
+
+function useUploader() {
+	const { actions: { uploadMedia } } = useAPI();
+	const { storyId, maxUpload, allowedMimeTypes: { image: allowedImageMimeTypes, video: allowedVideoMimeTypes } } = useConfig();
+	const allowedMimeTypes = [ ...allowedImageMimeTypes, ...allowedVideoMimeTypes ];
+
+	const isValidType = useCallback( ( { type } ) => {
+		return allowedMimeTypes.includes( type );
+	}, [ allowedMimeTypes ] );
+
+	const fileSizeCheck = useCallback( ( { size } ) => {
+		return size < maxUpload;
+	}, [ maxUpload ] );
+
+	const uploadFile = ( file ) => {
+		// TODO Add permission check here, see Gutenberg's userCan function.
+		if ( ! fileSizeCheck( file ) ) {
+			throw new Error( `File size error` );
+		}
+
+		if ( ! isValidType( file ) ) {
+			throw new Error( `File type error` );
```

Comment:
Nit: use `'` since no var is used within the string.

Reply:
Fixed in https://github.com/google/web-stories-wp/pull/195/commits/78d290c02621ddb81f4c2d552986a01f916a5d1c